src/sentry/api/endpoints/organization_projects.py

```diff
@@ -82,6 +82,18 @@ def get(self, request, organization):
                 organization=organization,
             ).prefetch_related('teams')
 
+        if request.user.is_authenticated():
+            queryset = queryset.extra(
+                select={
+                    'is_bookmarked': """exists (
```

Comment:
Does this even work on anything other than Postgres?

Reply:
It passes on Travis for sqllite and mysql, does that run the entire test suite?

Reply:
This will be fine in MySQL.